tests/app-test.js

```diff
@@ -1,10 +1,77 @@
-/* global describe, beforeEach, afterEach, it */
+/* global describe, beforeEach, afterEach */
 import { expect } from 'chai';
+import $ from 'jquery';
+import sinon from 'sinon';
+import it from './it-will';
```

Comment:
What about `it.immediately` for the original behavior. The reason I say is that we want "not thinking about asynchrony" to be the default.